index.html

```diff
@@ -33,19 +33,134 @@ <h2>I’am a Software Developer</h2>
+                <ul class="projects">
+                    <li class="list5"><a href="# ">html</a></li>
+                    <li class="list5"><a href="# ">bootstrap</a></li>
+                    <li class="list5"><a href="#">Ruby</a></li>
+                </ul>
+                <div class="action-bar">
+                    <a>see project</a>
+                </div>
+                <div class="projects3">
+                    <h1>Profesional Art Printing Data</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="projects">
+                        <li class="list5"><a href="# ">html</a></li>
+                        <li class="list5"><a href="# ">bootstrap</a></li>
+                        <li class="list5"><a href="#">Ruby</a></li>
+                    </ul>
+                    <div class="action-bar">
+                        <a>see project</a>
+                    </div>
+                </div>
             </div>
-            <div class="grid-item2"></div>
-            <div class="grid-item3"></div>
-            <div class="grid-item4"></div>
+            <div class="grid-item4">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="projects">
+                    <li class="list5"><a href="# ">html</a></li>
+                    <li class="list5"><a href="# ">bootstrap</a></li>
+                    <li class="list5"><a href="#">Ruby</a></li>
+                </ul>
+                <div class="action-bar">
+                    <a>see project</a>
+                </div>
+                <div class="projects4">
+                    <h1>Profesional Art Printing Data</h1>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="projects">
+                        <li class="list5"><a href="# ">html</a></li>
+                        <li class="list5"><a href="# ">bootstrap</a></li>
+                        <li class="list5"><a href="#">Ruby</a></li>
+                    </ul>
+                    <div class="action-bar">
+                        <a>see project</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="About">
+            <div class="aboutme">
+                <h1>About me</h1>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="resume">Get my Resume</div>
+
+
```

Comment:
- [ ] Please always try to pay attention to the design details 🥏, sticking to the details is what makes your whole portfolio stick out. As you see in the screenshots below you need to: `space your content in the center` `use the correct font for the text` and `use the correct color for the Get my Resume button` 👇🏽

### Figma design:
![image](https://user-images.githubusercontent.com/43172164/205433408-8887347b-4c82-4348-abdd-09be2bcc9a44.png)
### Your design:
![image](https://user-images.githubusercontent.com/43172164/205433421-95d78d72-7502-4d59-9a55-28642f904201.png)